packages/@okta/vuepress-site/docs/reference/api/sessions/index.md

```diff
@@ -736,14 +700,14 @@ The [authentication methods reference](https://tools.ietf.org/html/draft-ietf-oa
 
 ### IDP object
 
-Specifies the identity provider used to authentication the user.
+Specifies the identity provider that is used to authenticate the user
 
 | Property     | DataType                                                      | Nullable  | Unique  | Readonly | MinLength | MaxLength | Validation |
 | ------------ | ------------------------------------------------------------- | --------- | ------- | -------- | --------- | --------- | ---------- |
 | id           | String                                                        | FALSE     | FALSE   | TRUE     |           |           |            |
 | type         | `OKTA`, `ACTIVE_DIRECTORY`, `LDAP`, `FEDERATION`, or `SOCIAL` | FALSE     | FALSE   | TRUE     |           |           |            |
 
-> The `id` will be the org id if the type is `OKTA`; otherwise it will be the IDP instance id.
+> The `id` is the org ID if the type is `OKTA`. Otherwise it will be the IDP instance ID.
```

Comment:
this should be a note - and then the second sentence edit: Otherwise, it is the IdP instance ID.